BraveWallet/Preview Content/MockKeyringService.swift

```diff
@@ -80,10 +80,6 @@ class TestKeyringController: NSObject, BraveWalletKeyringController {
     completion(mnemonic)
   }
   
-  func defaultKeyringInfo(_ completion: @escaping (BraveWallet.KeyringInfo) -> Void) {
-    completion(keyring)
-  }
-  
   func keyringInfo(_ keyringId: String, completion: @escaping (BraveWallet.KeyringInfo) -> Void) {
     completion(keyring)
   }
```

Comment:
Not a big deal for now but we should update our mock keyring service to support multiple keyring IDs eventually

Reply:
I have updated and created a list of keyrings (currently with only one keyring in the list and it acts as the default keyring with default keyring id)